crates/biome_js_analyze/src/lint/nursery/use_date_now.rs

```diff
@@ -32,8 +32,8 @@ declare_rule! {
     /// const foo = new Date().valueOf();
     /// ```
     ///
-    /// ```js,expect_diagnostic,ignore
-    /// const foo = +new Date;
```

Comment:
Why this file was changed?

Reply:
See https://github.com/biomejs/biome/pull/3059 I started fixing documentation but noticed bug with imports so here we are.

I made mistake writing this example, useDateNow does not match `new Date` because it is not call expression